### PR TITLE
Model

### PR DESCRIPTION
**Pull request by nickelcheng**
- Add database migration into install.py.
- Create database tables for problem, contest, team, and users. (Tables for group isn't included.)

The database demo at http://140.114.77.129:8000/admin
demo / 123

The following is database design.
screenshot from 2015-01-30 00 16 59

### CODE REVIEW

File: install.py [hunk 1]
```diff
@@ -23,22 +23,34 @@
...
+
+
+# Database Migratinos
+os.system('python ./dbMigrate.py')
+
+
+# Create super user
+ans = raw_input('Create super user?[Y/n] ')
+if ans == '' or ans == 'y' or ans == 'Y':
+    os.system('python ./manage.py createsuperuser')
```
Comment (bruce3557) [inline install.py]: I prefer like write python with more function. That will make code easier to read.

File: problem/models.py [hunk 2]
```diff
@@ -1,3 +1,151 @@
...
+        (NOT_ACCEPTED, 'Not Accepted'),
+        (COMPILE_ERROR, 'Compile Error'),
+        (RESTRICTED_FUNCTION, 'Restricted Function'),
+        (JUDGE_ERROR, 'Judge Error'),
+    )
+    status = models.CharField(max_length=2, choices=STATUS_CHOICE, default=WAIT)
+
+    C = 'C'
+    CPP = 'CP'
+    CPP11 = '11'
```
Comment (bruce3557) [inline problem/models.py]: I prefer use full name to get clear naming rule.

File: team/models.py [hunk 3]
```diff
@@ -1,3 +1,60 @@
...
+
+    team = models.ForeignKey(Team)
+    member = models.ForeignKey(User)
+
+    VALID = 'V'
+    INVITED = 'I'
+    APPLY = 'A'
+    MEMBER_STATUS_CHOICE = (
+        (VALID, 'Valid'), (INVITED, 'Invited'), (APPLY, 'Apply'),
+    )
```
Comment (bruce3557) [inline team/models.py]: Same issue with language choice

File: users/models.py [hunk 4]
```diff
@@ -1,3 +1,74 @@
...
+    ADMIN = 'A'
+    JUDGE = 'J'
+    SUB_JUDGE = 'S'
+    USER = 'U'
+    USER_LEVEL_CHOICE = (
+        (ADMIN, 'Admin'),
+        (JUDGE, 'Judge'),
+        (SUB_JUDGE, 'Sub-judge'),
+        (USER, 'User'),
+    )
```
Comment (bruce3557) [inline users/models.py]: Ditto

File: users/models.py [hunk 5]
```diff
@@ -1,3 +1,74 @@
...
+    COSMO = 'C'
+    DEFAULT = 'D'
+    LUMEN = 'L'
+    THEME_CHOICE = (
+        (PAPER, 'Paper'),
+        (READABLE, 'Readable'),
+        (COSMO, 'Cosmo'),
+        (DEFAULT, 'Default'),
+        (LUMEN, 'Lumen'),
+    )
```
Comment (bruce3557) [inline users/models.py]: Ditto

**Comment by nickelcheng** [comment]
All issues fixed. Any other suggestion?

**Comment by bruce3557** [comment]
Great, going to merge.

**Pull request merged**